run_axelrod

```diff
@@ -1,135 +1,147 @@
-        action='store_true',
-        dest="exclude_ordinary",
-        help='exclude ordinary strategies plot')
-
-    parser.add_argument(
-        '--xc', "--exclude-cheating",
-        action='store_true',
-        dest="exclude_cheating",
-        help='exclude cheating strategies plot')
-
-    parser.add_argument(
-        '--xa', "--exclude-combined",
-        action='store_true',
-        dest="exclude_combined",
-        help='exclude combined strategies plot')
-
-    parser.add_argument(
-        '--ne', "--no-ecological",
-        action='store_true',
-        dest="no_ecological",
-        help='no ecological variant')
-
-    parser.add_argument(
-        '-p', '--processes',
-        type=int,
-        default=None,
-        help='Number of parallel processes to spawn. 0 uses cpu count.')
-
-    parser.add_argument(
-        '--rc', "--rebuild-cache",
-        action='store_true',
-        dest="rebuild_cache",
-        help='rebuild cache and save to file')
-
-    parser.add_argument(
-        '-c', '--cache_file',
-        type=str,
-        default='./cache.txt',
-        help='Path to cache file')
-
-    parser.add_argument(
-        '-n', '--noise',
-        type=float,
-        default=0,
-        help='Noise level')
-
-    parser.add_argument(
-        '-i', '--image_format',
-        type=str,
-        default="svg",
-        help='Image format for matplotlib, e.g. svg, jpeg, png')
-
-    args = parser.parse_args()
-
-    if all([args.exclude_basic,
-            args.exclude_ordinary,
-            args.exclude_cheating,
-            args.exclude_combined]):
-        print("You've excluded everything - nothing for me to do")
     else:
-        setup_logging(args.logging_destination, args.verbosity)
-        # Unravel argparse Namespace object to python keyword arguments.
-        kwargs = vars(args)
-        del kwargs["logging_destination"]
-        del kwargs["verbosity"]
-
-        if kwargs['prob_end'] is None:
-            del kwargs["prob_end"]
-            run_tournaments(**kwargs)
-        else:
-            del kwargs["turns"]
-            run_prob_end_tournaments(**kwargs)
+        tournament = axl.ProbEndTournament(players, repetitions=repetitions,
+                                           turns=turns, noise=noise,
+                                           processes=processes,
+                                           deterministic_cache=cache)
+        label = "Turns: {}, repetitions: {}, noise: {}, strategies: {}. ".format(turns, tournament.repetitions, noise, len(tournament.nplayers))
+
+    return tournament.play(), label
+
+
+if __name__ == '__main__':
+    arguments = docopt(__doc__, version='Run Axelrod 0.1')
+
+    # Parse arguments
+    turns = int(arguments['-t'])
+    repetitions = int(arguments['-r'])
+    prob_end = float(arguments['-e'])
+    noise = float(arguments['-n'])
+    processes = eval(arguments['-p'])
+
+    output_directory = arguments['-o']
+    if output_directory == "None":
```

Comment:
Yeah, this felt a bit icky, could get around it by changing the defaults to something like ` "" ` but that felt ickier...